.travis.yml

```diff
@@ -7,6 +7,7 @@ script:
   - crystal spec
   - crystal spec --release --no-debug
   - crystal tool format --check
+  - bin/ameba src
```

Comment:
Seems like the linter should go first to fail fast and not wast time on CI.

Reply:
Nope, I believe it's more important to see failing specs first.

Reply:
sounds good :)